ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixParticipant.java

```diff
@@ -91,6 +94,26 @@ public void initialize(String hostName, int port, List<AmbryHealthReport> ambryH
     }
   }
 
+  @Override
+  public void setReplicaSealedState(ReplicaId replicaId, boolean isSealed) {
+    HelixAdmin helixAdmin = manager.getClusterManagmentTool();
+    InstanceConfig instanceConfig = helixAdmin.getInstanceConfig(clusterName, instanceName);
+    List<String> sealedReplicas = ClusterMapUtils.getSealedReplicas(instanceConfig);
+    if (sealedReplicas == null) {
+      sealedReplicas = new ArrayList<>();
+    }
+    String partitionId = replicaId.getPartitionId().toPathString();
+
+    if (!isSealed) {
+      sealedReplicas.remove(partitionId);
+    }
+    else if (!sealedReplicas.contains(partitionId)) {
```

Comment:
looks like an inconsistency with the formatting guidelines. Did you format the file after changes? (unfortunately does not happen automatically)

Reply:
fixed